Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -110,14 +109,13 @@ public async Task<Loc> GetLocationFromDevice()
         /// <returns>Boolean Value</returns>
         public async Task<bool> Register()
         {
-            await ConfigureMobiledgeXSettings(LocationNeeded: false);
-
+            await ConfigureMobiledgeXSettings();
+            me.SetMelMessaging(new MelMessaging(appName));
```

Comment:
Hi, As we just discussed. Init once, the appName shouldn't change during runtime. MelMessaging happens to be mostly static, but that depends on the class loader if that's true. Should move out to some Init outside.

Reply:
Done, MelMessaging Initialization is in the MobiledgeXIntegration Constructor